components/automate-builder-api/habitat/hooks/run

```diff
@@ -22,6 +22,7 @@ render-template config.toml "{{pkg.svc_var_path}}/etc/config.toml" \
 export DBNAME="{{cfg.datastore.database}}"
 pg-helper ensure-service-database "$DBNAME"
 pg-helper fix-permissions "$DBNAME"
+pg-helper alter-role --superuser=true "$DBNAME"
```

Comment:
We should alter the database `user` role . User can be obtained from `cfg.datastore.user`